src/amo/pages/Addon/index.js

```diff
@@ -159,7 +159,13 @@ export class AddonBase extends React.Component {
     const { addon, i18n } = this.props;
 
     if (addon && isTheme(addon.type)) {
-      return <ThemeImage addon={addon} roundedCorners />;
+      return (
+        <ThemeImage
+          addon={addon}
+          index={getPreviewIndexBySize(addon)}
```

Comment:
Similar to my other comment, how about making this `preferWidest: boolean` ? I don't know if that's the right name but the idea is to hide the logic of `getPreviewIndexBySize()` in the implementation of `ThemeImage`.

Reply:
yeh it's not the widest but I'll take another look 👀

Reply:
It could be `prefer720px` but whatever you think makes the most sense

Reply:
Maybe not `720px` because we already know the size changed once

Reply:
Yeah, that sounds good

Reply:
cool:)